### ISSUE TITLE
Deprecated config in `setup.cfg` (Python 3.12)

### ISSUE THREAD

**Issue by ArchLeaders** [zeldamods/event-editor#12]
I get this error when running `pip install eventeditor` on a fresh install of Python 3.12.

```
C:\Users\ArchLeaders\AppData\Local\Temp\pip-build-env-ivyrswtq\overlay\Lib\site-packages\setuptools\config\setupcfg.py:293: _DeprecatedConfig: Deprecated config in `setup.cfg`
```
Entire Output: [Output.txt](https://github.com/zeldamods/event-editor/files/13236460/Output.txt)

**Comment by VillainousSsnake** [comment]
It seems to use the oead library, so try using Python 3.11.7

**Comment by leoetlino** [comment]
Should be fixed by 1.3.9-1

**Comment by VillainousSsnake** [comment]
Oo nice
